proxy/api/src/identity.rs

```diff
@@ -64,14 +64,22 @@ pub async fn get(state: &coco::State, id: coco::Urn) -> Result<Identity, error::
     Ok((state.peer_id(), user).into())
 }
 
+// TODO(finto): Check if this is used and if so, express more elegantly after
```

Comment:
This isn't used anywhere yet. It corresponds to the `GET identities/` endpoint. @MeBrei this was for the list of profiles work, right?